habitat-baselines/habitat_baselines/rl/ver/README.md

```diff
@@ -13,7 +13,7 @@ The system has 5 components. 3 of these are the key components for learning:
 
 How these pieces are connected together is best seen via this diagram:
 
-![ver System Diagram](/habitat_baselines/rl/ver/images/ver-system.svg)
+![ver System Diagram](images/ver-system.svg)
```

Comment:
svg does not work I believe. We might have to export to PNG

Reply:
Ignore that comment, fixed it in [287bf45](https://github.com/facebookresearch/habitat-lab/pull/1000/commits/287bf45b51ba6266bb3d3149300d0ac38fe5c540)